pkg/storaged/utils.js

```diff
@@ -663,48 +663,48 @@
                 return as[0] == block.path;
             });
             usage.push({
-                level: level,
+                level,
                 usage: 'mdraid-member',
-                block: block,
-                mdraid: mdraid,
+                block,
+                mdraid,
                 location: mdraid_name(mdraid.Name),
                 actions: get_actions(_("remove from RAID")),
                 blocking: !(active_state && active_state[1] < 0)
             });
         } else if (vgroup) {
             usage.push({
-                level: level,
+                level,
                 usage: 'pvol',
-                block: block,
-                vgroup: vgroup,
-                pvol: pvol,
+                block,
+                vgroup,
+                pvol,
                 location: vgroup.Name,
                 actions: get_actions(_("remove from LVM2")),
                 blocking: pvol.FreeSize != pvol.Size
             });
         } else if (vdo) {
             usage.push({
-                level: level,
+                level,
                 usage: 'vdo-backing',
-                block: block,
-                vdo: vdo,
+                block,
+                vdo,
                 location: vdo.name,
                 blocking: true
             });
         } else if (stratis_pool) {
             usage.push({
-                level: level,
+                level,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18417-20230227-210817-602a6f78-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)